schemas/2014-04-01-preview/Microsoft.Sql.json

```diff
@@ -169,6 +170,54 @@
         "properties"
       ],
       "description": "Microsoft.Sql/firewallrules"
+    },
+    "administratorsChild": {
+      "type": "object",
+      "properties": {
+        "type": {
+          "enum": [
+            "administrators"
+          ]
+        },
+        "apiVersion": {
+          "enum": [
+            "2014-04-01-preview"
+          ]
+        },
+        "properties": {
+          "$ref": "#/definitions/administratorsBase"
+        }
+      },
+      "required": [
+        "type",
+        "apiVersion",
+        "properties"
+      ],
+      "description": "Microsoft.Sql/administratorsChild"
+    },
+    "administrators": {
```

Comment:
Please update in 2015-01-01/deploymentTemplate.json and 2014-04-01-preview/deploymentTemplate.json, so that "adminstrators" can be referenced.